### ISSUE TITLE
Tasks

### ISSUE THREAD

**Issue by pgjones** [pgjones/quart-schema#20]
- [x] Stop altering the JSONProvider (do casing directly with dicts)
- [ ] Document how to order the openapi schema routes
- [ ] Document how to customise the openapi schema

**Comment by larsblumberg** [comment]
Hi @pgjones, seeing that you plan to document how to customize the openapi schema, I hope I don't hijack this issue when I ask about how we could add a custom documentation/description string to each view/route?

**Comment by pgjones** [comment]
The view/route docstring should be added to the openapi schema - is that what you want?

**Comment by larsblumberg** [comment]
> The view/route docstring should be added to the openapi schema - is that what you want?

Yes, having docstring added to the schema would be great. And now that you ask I figure that the docstring is already used for the openapi schema generation: https://github.com/pgjones/quart-schema/blob/50e8a382af554e5d227b34f427520946549eafe1/src/quart_schema/extension.py#L432

**Comment by Beetroit** [comment]
@pgjones any updates on ordering the schema routes?

**Comment by pgjones** [comment]
@Beetroit best to override the `generate_rules` method and specify the sorting.
